src/langserver/Cargo.toml

```diff
@@ -5,14 +5,15 @@ authors = ["Tad Hardesty <tad@platymuus.com>"]
 edition = "2018"
 
 [[bin]]
-name = "dm-langserver"
+name = "dm_langserver"
```

Comment:
this was changed to avoid a rust bug where symbols are not generated (at least on Windows with MSVC) if a dash is in the name

Reply:
is there an upstream ticket?

Reply:
for this specific issue idk, it was a problem before but got 'fixed'.
https://github.com/rust-lang/cargo/pull/8123
perhaps it was only fixed when there was no `[[bin]].name` override.

Reply:
I will probably revert this in the name of not having to modify my deploy scripts, but I suggest filing an up-to-date report upstream

Reply:
That's fine. It's more that I forgot to remove it first, but it was worth pointing out anyway.